cabot/cabotapp/views.py

```diff
@@ -726,3 +726,52 @@ def get(self, request, *args, **kwargs):
 @register.filter
 def get_item(dictionary, key):
     return dictionary.get(key)
+
+#===============================================================================
+# Activity counter functions
+#===============================================================================
+
+def json_response(data, code):
+    '''Return a JSON response containing some data'''
+    return HttpResponse(json.dumps(data), status=code, content_type="application/json")
+
+def json_error_response(message, code):
+    '''Return a JSON response with an error message'''
+    return json_response({ 'detail': message }, code)
+
+class ActivityCounterView(View):
```

Comment:
Would be good to add unit tests for this ([APITestCase](http://www.django-rest-framework.org/api-guide/testing/#api-test-cases) may be helpful. it's also used in some Cabot tests [here](https://github.com/Affirm/cabot/blob/master/cabot/cabotapp/tests/tests_basic.py#L35))

Reply:
Yeah, I forgot about tests completely.  Will add them!